cmd/sedna-lc/app/server.go

```diff
@@ -86,17 +92,17 @@ func runServer() {
 		return
 	}
 
-	dm := manager.NewDatasetManager(c, Options)
+	dm := dataset.NewDatasetManager(c, Options)
 
-	mm := manager.NewModelManager(c)
+	mm := model.NewModelManager(c)
```

Comment:
```suggestion
	mm := model.New(c)
```

Reply:
Since the package `model` has already model context, no need NewModelManager

Reply:
fix it